package.json

```diff
@@ -23,9 +23,12 @@
     "prettier": "^2.1.2"
   },
   "devDependencies": {
-    "danger": "7.0.0",
+    "danger": "^8.0.0",
     "danger-plugin-digitalcitizenship": "*",
     "ts-node": "^9.1.1",
     "typescript": "^4.1.3"
+  },
+  "resolutions": {
+    "gitlab": "6.3.7"
```

Comment:
What is this for?

Reply:
This is to avoid `TypeError: global$1.XMLHttpRequest is not a constructor` error during danger execution ([link](https://dev.azure.com/pagopa-io/eslint-rules/_build/results?buildId=8757&view=logs&j=6370a2a0-b716-5dad-4f74-710354e6817b&t=4ce6d61b-8fc4-588b-2698-2f5422af8b08)).
Some reference found in [here](https://github.com/danger/danger-js/issues/884#issuecomment-507282808)

Reply:
Alternatively, we can use the same solution proposed by @GiovanniMancini upgrading to `danger@10.xx`

Reply:
> upgrading to danger@10.xx

Is it hard to migrate?

Reply:
I think that only package upgrade is enough (no other changes are needed).

Reply:
fixed in c0de0e2